.github/workflows/comments.yml

```diff
@@ -0,0 +1,34 @@
+# Communicates with contributors using comments
+#
+# New PRs receive a welcome and some direction
+# Merges receive a pointer to any post-merge rollouts
+#
+name: PR Comments
+
+on:
+  pull_request:
+    types: [opened, reopened, synchronize, closed]
+
+jobs:
+  variables:
+    name: Comment on New Pull Request
+    runs-on: ubuntu-latest
+    steps:
+      - name: Comment on PR
+        if: github.event.pull_request.merged == false && github.event.pull_request.comments == 0
+        uses: thollander/actions-comment-pull-request@master
+        with:
+          message: >-
+            Thanks for the pull request! <br>
+            Please see [CONTRIBUTING.md](https://github.com/bcgov/wps-web/blob/master/CONTRIBUTING.yaml) 
+            for details.
+          GITHUB_TOKEN: ${{ secrets.REPO_ACCESS_TOKEN }}
+
+      - name: Comment on PR
+        if: github.event.pull_request.merged == true
+        uses: thollander/actions-comment-pull-request@master
```

Comment:
@Kyubinhan @DerekRoberts @andrea-williams something to ruminate on: using github actions from various repos - it implies a level of trust. I think we need to educate ourselves on the implications - a quick google will take you down a rabbit hole: https://julienrenaux.fr/2019/12/20/github-actions-security-risk/